<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CExchange</class>
 <widget class="QDialog" name="CExchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exchange</string>
  </property>
  <layout class="QVBoxLayout" name="pLayout">
   <item>
    <widget class="QComboBox" name="pCBCurrency1"/>
   </item>
   <item>
    <widget class="QLineEdit" name="pLEAmount1"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="pLERate"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBCalc">
       <property name="text">
        <string>Calc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="pLCurrency2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Currency2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pLEAmount2">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="pButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pCBCurrency1</tabstop>
  <tabstop>pLEAmount1</tabstop>
  <tabstop>pLERate</tabstop>
  <tabstop>pPBCalc</tabstop>
  <tabstop>pButtonBox</tabstop>
  <tabstop>pLEAmount2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CExchange</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CExchange</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
